Cargo.toml

```diff
@@ -20,7 +20,7 @@ nb = "0.1"
 numtoa = "0.2"
 
 [dev-dependencies]
-embedded-hal-mock = "0.7"
+embedded-hal-mock = { git = "https://github.com/dbrgn/embedded-hal-mock/", branch = "serial-error-handling" }
```

Comment:
Release will have to wait until https://github.com/dbrgn/embedded-hal-mock/pull/32 is merged and released.

Reply:
Released in 0.7.2!